docs/packages.txt

```diff
@@ -1,3 +1,4 @@
 opkg install pip3
 pip3 install scapy
-pip3 install mitmproxy
+pip3 install python-nmap
```

Comment:
Please update tinyHIPPOstart as well with the following:
`pip3 install python-nmap`